index.js

```diff
@@ -60,7 +134,28 @@
  * };
  *
  */
+ var = allFunctions {
+ 	q0: countdown(),
+ 	q1: max(),
+ 	q2: longer(),
+ 	q3: isEven(),
+ 	q4: getAreaOfCircle(),
+ 	q5: getFullName(),
+ 	q6: reverseString(),
+ 	q7: maxArray()
+ };
 
  /**
   * Q9: Use the Object from Q8 to call all of the functions you've created.
   */
+  var count = 0;
+  var i;
+
+  function callFunctions() {
+	for (i in allFunctions) {
```

Comment:
Here you are using a `for` `in` loop, which we have not discussed at all in class to loop through an Object's keys. This is a typical way to iterate through properties in an Object, which isn't something we've really touched on so far in class.